magellan-library/src/main/java/com/wealthfront/magellan/view/ActionBarModifier.java

```diff
@@ -27,6 +27,13 @@ default boolean shouldAnimateActionBar() {
     return true;
   }
 
+  /**
+   * @return true if we need to pass the onUpdateMenu calls to the navigables children, false otherwise (false by default).
+   */
+  default boolean shouldShowChildNavigablesMenu() {
+    return false;
+  }
```

Comment:
Remind me again how we propagate `onUpdateMenu` and populate the menu? Is it in `onUpdateMenu` above, i.e. the implementation is provided by the implementor of this interface?

This method is kind of weird with no implementation backing it. `shouldShowActionBar` above is acted on by another object, so that's exposing this state to another class. But overriding this method and expecting the inheritor of this class to respect it is a strange pattern.

Maybe we should add this to `LegacyScreen` or wherever the logic for passing the menu to its children actually lives.

Reply:
Remove this.